src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/java/util/WmsRequestInterceptor.java

```diff
@@ -0,0 +1,55 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package}.util;
+
+import de.terrestris.shogun2.util.interceptor.MutableHttpServletRequest;
+import de.terrestris.shogun2.util.interceptor.WmsRequestInterceptorInterface;
+
+/**
+ * This class demonstrates how to implement the WmsRequestInterceptorInterface.
+ * 
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class WmsRequestInterceptor implements WmsRequestInterceptorInterface {
```

Comment:
Couldn't this trivial implementation be a basic/default implementation that is located in SHOGun, but not in the archetype?